src/pages/resultsView/coExpression/CoExpressionViz.tsx

```diff
@@ -262,18 +276,21 @@ export default class CoExpressionViz extends React.Component<ICoExpressionVizPro
     }
 
     @computed get showLogScaleControls() {
-        const profileId = this.props.molecularProfile.molecularProfileId.toLowerCase();
+        const profileXId = this.props.profileX.molecularProfileId.toLowerCase();
+        const profileYId = this.props.profileY.molecularProfileId.toLowerCase();
         return (
-            (profileId.indexOf("rna_seq") > -1) &&
-            (profileId.indexOf("zscore") === -1)
+            ((profileXId.indexOf("rna_seq") > -1) &&
+            (profileXId.indexOf("zscore") === -1)) ||
+            ((profileYId.indexOf("rna_seq") > -1) &&
+            (profileYId.indexOf("zscore") === -1))
         );
     }
 
     private get requestAllDataButton() {
         if (this.plotData.isComplete && !this.dataStore.allData.length && !this.allDataRequested) {
```

Comment:
please leave all the rendering functions that depend on `somepromise.isComplete` alone (as you've done) - it's just the @computed's that we want to change